common/src/View/MapDocumentCommandFacade.cpp

```diff
@@ -937,7 +950,11 @@ namespace TrenchBroom {
 
             // to avoid backslashes being misinterpreted as escape sequences
             const std::string formatted = kdl::str_replace_every(spec.asString(), "\\", "/");
-            m_world->addOrUpdateAttribute(Model::AttributeNames::EntityDefinitions, formatted);
+
+            auto entity = m_world->entity();
+            entity.addOrUpdateAttribute(Model::AttributeNames::EntityDefinitions, formatted);
```

Comment:
This fragment is copy/pasted enough it might be worth having a helper function in EntityNode,

Reply:
I have considered it, but I actually like this to be a bit more expressive so that you know what's going on under the hood, and that you should group multiple modifications to the entity object together instead of calling the helper function multiple times.

Reply:
Right, that makes sense